bitnami/zookeeper/Chart.yaml

```
@@ -15,4 +15,4 @@ name: zookeeper
 sources:
   - https://github.com/bitnami/bitnami-docker-zookeeper
   - https://zookeeper.apache.org/
-version: 6.0.1
+version: 6.0.2
```

As this includes a new parameter, please bump the minor version.

```suggestion
version: 6.1.0
```